autogluon/training/docker/0.3.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,135 @@
+ARG FROM_IMAGE=ubuntu:18.04
+FROM ${FROM_IMAGE}
+LABEL maintainer="Amazon AI"
+
+ARG PYTHON_VERSION=3.7.10
+ARG PYTHON=python3
+ARG MXNET_VER=1.7.*
+ARG TORCH_VER=1.8.0+cpu
+ARG TORCH_VISION_VER=0.9.0+cpu
+
+# Update once 0.3.0 is released
+ARG AUTOGLUON_VERSION=0.2.1b20210625
```

Comment:
I'd suggest to put **this** file into 0.2/py3/Dockerfile.cpu directory. And create a new docker file in 0.3/py3/ as soon as you have 0.3.0 released.

Reply:
Changed to 0.2.1